ephios/extra/management/commands/send_notifications.py

```diff
@@ -0,0 +1,24 @@
+import traceback
+
+from django.core.mail import mail_admins
+from django.core.management import BaseCommand
+
+from ephios.core.models.users import Notification
+from ephios.core.notifications.backends import all_notification_backends
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        for backend in all_notification_backends():
+            for notification in Notification.objects.filter(failed=False):
+                if backend.can_send(notification) and backend.user_prefers_sending(notification):
+                    try:
+                        backend.send(notification)
+                    except Exception as e:
```

Comment:
pylint doesn't allow this, but I don't know what to use here

Reply:
my thought was that a lot could happen until the notification gets send out, e.g. the event could have been deleted which would raise Event.DoesNotExist.
what would you suggest to handle this?

Reply:
It's probably good to catch all exceptions, but we should log them.